101-anf-smb-volume/prereqs/azuredeploy.json

```diff
@@ -0,0 +1,408 @@
+{
```

Comment:
As stated in another comment, we should clean up the the content of this quick start template related to AD items.

Reply:
as stated in the previous comment, I copied all the files as a prerequisites. I am trying to find a way to reference to the other quickstart without copying the files again.

Reply:
Changed into a module